### PR TITLE
Update chromium to 65.0.3325.146

### PR DESCRIPTION
**Pull request by squalus**
- update debian patches from 65.0.3325.85-1 branch
- update inox patches from 65.0.3325.146 branch
- remove obsolete fix-icu-tangut patch (the bad symbols don't appear to be there anymore)
- fix http status symbol build error in disable-webrtc-log-uploading patch
- refresh other patches referenced from the config bundles, excluding windows
- add extra binary to the pruning exclusion list due to build error

Tested a linux_simple build on gentoo ~amd64 and it seems ok to me using it for a few hours. Patches for macOS and other Linux configs were refreshed but untested. Windows patches weren't touched at all.

### CODE REVIEW

**Comment by Eloston** [comment]
Thanks! I'll merge this after I merge xsmile's PR for Arch Linux (which should be after the build is confirmed working).

File: resources/patches/debian/system/gtk2.patch [hunk 1]
```diff
@@ -1,13 +0,0 @@
-description: add missing include needed to build with gtk2
```
Comment (Eloston) [inline resources/patches/debian/system/gtk2.patch]: Was the GTK2 patch removed in Debian?

File: developer_utilities/update_lists.py [hunk 2]
```diff
@@ -34,6 +34,7 @@
 # pathlib.Path.match() paths to exclude from binary pruning
 PRUNING_EXCLUDE_PATTERNS = [
     'chrome/common/win/eventlog_messages.mc', # TODO: False positive textfile
+    'chrome/browser/resources/media/mei_preload/preloaded_data.pb',
```
Comment (Eloston) [inline developer_utilities/update_lists.py]: Do you know what this file is used for?

File: resources/patches/debian/system/gtk2.patch (hunk 1 above)
Comment (squalus) [reply resources/patches/debian/system/gtk2.patch]: Yeah. It seems they fixed the gtk2 build error upstream

File: developer_utilities/update_lists.py (hunk 2 above)
Comment (squalus) [reply developer_utilities/update_lists.py]: Not really. MEI = media engagement index, which is about collecting analytics in order to determine whether to allow videos to auto-play

It does break the build if not included. It's only 14 bytes and doesn't contain anything interesting if printed with `protoc --decode_raw`. 

```
media/base/media_switches.cc
// Allows Media Engagement to use preloaded data to decide whether an origin has
// a high media engagement.

// - PreloadMediaEngagementData: enables a list of origins to be considered as
//   having a high MEI until there is enough local data to determine the user's
//   preferred behaviour.

```

Maybe in an official branded build they preload a larger file with real data in it?

Comment (squalus) [reply developer_utilities/update_lists.py]: The proto is at `chrome/browser/media/media_engagement_preload.proto`

```
message PreloadedData {
  // The data in DAFSA format.
  optional bytes dafsa = 1;
}
```

Some more references:
`chrome/browser/media/media_engagement_preloaded_list.cc`
`chrome/browser/media/media_engagement_session.cc`
 `net/tools/dafsa`

**Comment by squalus** [comment]
macOS is now tested and working for me

File: developer_utilities/update_lists.py (hunk 2 above)
Comment (xsmile) [reply developer_utilities/update_lists.py]: Alternatively the mei_preload files can be excluded with this patch:
```
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -304,7 +304,6 @@ if (!is_android && !is_mac) {
       }
 
       data_deps += [
-        "//chrome/browser/resources/media/mei_preload:component",
         "//third_party/widevine/cdm:widevinecdmadapter",
       ]
 
@@ -1239,7 +1238,6 @@ if (is_win) {
       ":widevine_cdm_library",
       "//build/config:exe_and_shlib_deps",
       "//chrome/app/nibs:chrome_xibs",
-      "//chrome/browser/resources/media/mei_preload:component_bundle",
     ]
 
     if (is_chrome_branded) {
```
I didn't notice anything unusual with such a build yet.

File: resources/config_bundles/linux_rooted/patch_order.list [hunk 3]
```diff
@@ -1,17 +1,14 @@
-inox-patchset/chromium-vaapi-move.patch
-inox-patchset/chromium-vaapi-init.patch
-inox-patchset/chromium-vaapi-rgbx.patch
+inox-patchset/chromium-vaapi-init-r16.patch
 inox-patchset/chromium-vaapi-r16.patch
 
 debian/master-preferences.patch
 debian/system/icu.patch
 debian/system/vpx.patch
```
Comment (xsmile) [inline resources/config_bundles/linux_rooted/patch_order.list]: The vpx and lcms2 patches can also be removed for Chromium 65, see the [Debian series file](https://sources.debian.org/src/chromium-browser/65.0.3325.146-1/debian/patches/series/).

Comment (squalus) [reply resources/config_bundles/linux_rooted/patch_order.list]: You're right, I'll remove them

File: developer_utilities/update_lists.py (hunk 2 above)
Comment (squalus) [reply developer_utilities/update_lists.py]: I prefer excluding these files too

Comment (Eloston) [reply developer_utilities/update_lists.py]: Yeah, I don't see much benefit of including them in this case.

@squalus Could you include @xsmile's change as a new patch in the `common` base bundle (and also refresh any subsequent patches that also touch this file)?

Comment (Eloston) [reply developer_utilities/update_lists.py]: Nevermind I can merge now and do it.

**Pull request merged**